app/components/common/Navigation.vue

```diff
@@ -464,7 +464,8 @@ export default Vue.extend({
                   li.dropdown.dropdown-hover
                     a.text-p(:href="isWideScreen ? navItem.url : null" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false")
                       span {{ $t(navItem.title) }}
-                      caret.dropdown-caret
+                      caret.dropdown-caret(v-if="/^\\/(league|play\\/ladder)/.test(document.location.pathname)" color="white")
+                      caret.dropdown-caret(v-else color="black")
```

Comment:
this could work but i prefere to use CSS `.dark-mode` to toggle color. since we may have other places beside league or play/ladder to have dark theme. you can check line 546 in this file

Reply:
how will you toggle svg color based on dark-mode class as we will need to pass different value to SVG vue component?

Reply:
anyway we can use js to detect css class and toggle svg color. can checking if css can handle it directly

Reply:
hmm, when I use js to detect the css 'dark-mode' class, it seems to run before css is applied so it doesn't work. I'm not sure how to do this.

Reply:
emmm what? do you run detect in mounted or created ?

Reply:
I am running it in computed

Reply:
oh, do I run everything in mounted and do everything in js?